javascript/packages/eslint-config-nebenan/test/valid/misc.jsx

```diff
@@ -19,7 +21,15 @@ const Empty = () => <div />;
 
 class RenderRequired extends React.Component {
   render() {
-    this.content = (<div />);
+    const { name } = this.props;
+
+    console.log(RenderRequired.propTypes);
```

Comment:
this was failing? why is this added?

Reply:
Newer eslint-plugin-react versions have a new rule that notifies users if `propTypes` is used in a problematic way.

https://github.com/goodhood-eu/styleguide/pull/15/files#diff-ec0faea0dc800eaf1e667c3b49d11ed8R32

Since we use it every now and than, I disabled that rule and added a test case

Reply:
i think we should instead clean those places up and enable the rule, please create a ticket

Reply:
I personally think it makes a lot of sense using PropTypes when passing through cleanProps. Don't really see the point in removing PropTypes at all

Reply:
it would affect bundle sizes quite a lot

Reply:
Ok makes sense.

Reply:
https://favro.com/organization/aee06c3fe6503c804f507dd9/22c630cc02b09f2d6f98b8b3?card=Goo-31500